data/text/pokemon_news.inc

```diff
@@ -0,0 +1,165 @@
+
+gUnknown_0828D0F8:: @ 0828D0F8
```

Comment:
Why are these unnamed and separate from the other TV strings?

Reply:
1) I haven't gotten around to naming them.
2) They're separated from the other TV strings by a size-0x90F incbin, same as in Ruby.